pkgs/games/xsokoban/default.nix

```diff
@@ -15,14 +15,18 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
+  prePatch = ''
+    substituteInPlace Makefile.in --replace 4755 0755
+  '';
+
   preConfigure = ''
     sed -e 's/getline/my_getline/' -i score.c
 
     chmod a+rw config.h
     cat >>config.h <<EOF
     #define HERE "@nixos-packaged"
     #define WWW 0
-    #define OWNER "'$(whoami)'"
+    #define OWNER "$(whoami)"
```

Comment:
Running `whoami` isn't a good idea at all, is it's an attempt for non-determinism.